packages/fluent-editor/src/table/modules/table-operation-menu.ts

```
@@ -356,21 +356,26 @@ export default class TableOperationMenu {
       'top': `${topPos}px`,
       'min-height': `${MENU_MIN_HEIGHT}px`,
+      'max-height': `${winHeight - topPos}px`,
       'width': `${MENU_WIDTH}px`,
+      'overflow-y': 'auto',
     }
-
-    if (menuHeight + top > winHeight && menuHeight < winHeight) {
+    // fix: 处理菜单超出屏幕
+    if (menuHeight + top > winHeight || topPos > winHeight / 2) {
       delete cssContent.top
+      cssContent['max-height'] = `${winHeight - 20}px`
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)